client/src/components/OverviewComponent/index.js

```diff
@@ -17,11 +16,12 @@ const OverviewComponent = ({
   isLeftImg,
   isHomePage,
   isDark,
+  isRed,
```

Comment:
also here you may write a descriptive name 
```suggestion
  backgroundIsRed,
```
also isDark >>>  titleIsDark